### PR TITLE
Style/IndentationConsistency

### PR DESCRIPTION
**Pull request by ShinjiTanimoto**

### CODE REVIEW

File: app/models/job/task.rb [hunk 1]
```diff
@@ -44,50 +44,50 @@ def execute
...
-        job_log.closed = Time.zone.now
-        Rails.logger.info("Completed Job #{id} in #{time * 1000} ms")
-      ensure
-        Job::TaskLogger.detach(job_log)
-        job_log.save
+      time = Benchmark.realtime do
+        ret = yield
       end
-      ret
+    rescue Exception => e
```
Comment (sunny4381) [inline app/models/job/task.rb]: __[rails_best_practices]__
* Don't rescue Exception [:ref](http://rails-bestpractices.com/posts/2012/11/01/don-t-rescue-exception-rescue-standarderror/)

File: app/controllers/concerns/ss/ajax_file_filter.rb [hunk 2]
```diff
@@ -6,43 +6,43 @@ module SS::AjaxFileFilter
...
-      super
-    end
+  def append_view_paths
+    append_view_path "app/views/ss/crud/ajax_files"
+    super
+  end
 
-    def select_with_clone
-      set_item
+  def select_with_clone
```
Comment (sunny4381) [inline app/controllers/concerns/ss/ajax_file_filter.rb]: __[rails_best_practices]__
* replace complex creation with factory method (item attribute_assignment_count > 2) [:ref](http://rails-bestpractices.com/posts/2010/07/21/replace-complex-creation-with-factory-method/)

File: app/controllers/concerns/history/log_filter.rb [hunk 3]
```diff
@@ -2,21 +2,21 @@ module History::LogFilter
...
 
   private
-    def put_history_log
-      log = History::Log.new
-      log.url          = request.path
-      log.controller   = params[:controller]
-      log.action       = params[:action]
-      log.user_id      = @cur_user.id if @cur_user
-      log.site_id      = @cur_site.id if @cur_site
+  def put_history_log
```
Comment (sunny4381) [inline app/controllers/concerns/history/log_filter.rb]: __[rails_best_practices]__
* replace complex creation with factory method (log attribute_assignment_count > 2) [:ref](http://rails-bestpractices.com/posts/2010/07/21/replace-complex-creation-with-factory-method/)
* use model association (for log) [:ref](http://rails-bestpractices.com/posts/2010/07/19/use-model-association/)

File: app/controllers/cms/agents/tasks/pages_controller.rb [hunk 4]
```diff
@@ -5,93 +5,93 @@ class Cms::Agents::Tasks::PagesController < ApplicationController
...
+    ids.each do |id|
+      @task.count
+      page = Cms::Page.site(@site).or(cond).where(id: id).first
+      next unless page
+      @task.log page.full_url
+      release_page page.becomes_with_route
+    end
+  end
+
+  def release_page(page)
```
Comment (sunny4381) [inline app/controllers/cms/agents/tasks/pages_controller.rb]: __[rails_best_practices]__
* replace complex creation with factory method (page attribute_assignment_count > 2) [:ref](http://rails-bestpractices.com/posts/2010/07/21/replace-complex-creation-with-factory-method/)

File: app/controllers/sys/site_copy_controller.rb [hunk 5]
```diff
@@ -6,63 +6,63 @@ class Sys::SiteCopyController < ApplicationController
...
-    def run
-      set_item
-      @item.attributes = get_params
-      @item.state = 'ready'
-      @item.started = nil
-      @item.closed = nil
-      @item.logs = []
-      if @item.save
-        Sys::SiteCopyJob.perform_later
+  def run
```
Comment (sunny4381) [inline app/controllers/sys/site_copy_controller.rb]: __[rails_best_practices]__
* replace complex creation with factory method (@item attribute_assignment_count > 2) [:ref](http://rails-bestpractices.com/posts/2010/07/21/replace-complex-creation-with-factory-method/)

File: app/controllers/uploader/files_controller.rb [hunk 6]
```diff
@@ -1,169 +1,169 @@
...
+    if @directory
+      create_directory
+    elsif @files
+      create_files
+    else
+      raise "400"
+    end
+  end
+
+  def update
```
Comment (sunny4381) [inline app/controllers/uploader/files_controller.rb]: __[rails_best_practices]__
* replace complex creation with factory method (@item attribute_assignment_count > 2) [:ref](http://rails-bestpractices.com/posts/2010/07/21/replace-complex-creation-with-factory-method/)

File: app/controllers/ezine/agents/nodes/form_controller.rb [hunk 7]
```diff
@@ -7,49 +7,49 @@ class Ezine::Agents::Nodes::FormController < ApplicationController
...
-      @entry.entry_type = entry_type
-      @entry.in_data = nil if @entry.entry_type == "delete"
-
-      if @entry.save
-        @entry_type_string = t("ezine.entry_type.#{entry_type}")
-        render action: :confirm
-      else
-        render action: entry_type.to_sym
-      end
+  def confirm
```
Comment (sunny4381) [inline app/controllers/ezine/agents/nodes/form_controller.rb]: __[rails_best_practices]__
* replace complex creation with factory method (@entry attribute_assignment_count > 2) [:ref](http://rails-bestpractices.com/posts/2010/07/21/replace-complex-creation-with-factory-method/)

File: app/controllers/rdf/vocabs_controller.rb [hunk 8]
```diff
@@ -10,67 +10,67 @@ class Rdf::VocabsController < ApplicationController
...
 
-    def save_file
-      temp_file = SS::TempFile.new
-      temp_file.cur_user = @cur_user
-      temp_file.in_file = @params.in_file
-      temp_file.state = "private"
-      temp_file.save!
-      temp_file
-    end
+  def save_file
```
Comment (sunny4381) [inline app/controllers/rdf/vocabs_controller.rb]: __[rails_best_practices]__
* replace complex creation with factory method (temp_file attribute_assignment_count > 2) [:ref](http://rails-bestpractices.com/posts/2010/07/21/replace-complex-creation-with-factory-method/)

File: app/controllers/opendata/agents/nodes/app/appfile_controller.rb [hunk 9]
```diff
@@ -6,43 +6,43 @@ class Opendata::Agents::Nodes::App::AppfileController < ApplicationController
...
 
-    def download
-      @item = @app.appfiles.find_by id: params[:id], filename: params[:filename].force_encoding("utf-8")
+  def download
+    @item = @app.appfiles.find_by id: params[:id], filename: params[:filename].force_encoding("utf-8")
 
-      send_file @item.file.path, type: @item.content_type, filename: @item.filename,
-        disposition: :attachment, x_sendfile: true
-    end
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/app/appfile_controller.rb]: __[brakeman]__
* Model attribute used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/opendata/dataset/resources_controller.rb [hunk 10]
```diff
@@ -10,95 +10,95 @@ class Opendata::Dataset::ResourcesController < ApplicationController
...
+    @item = @dataset.resources.find params[:resource_id]
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
+      disposition: :attachment, x_sendfile: true
+  end
+
+  def download_tsv
+    @item = @dataset.resources.find params[:resource_id]
+    raise "404" if @item.tsv.blank?
+    raise "403" unless @dataset.allowed?(:read, @cur_user, site: @cur_site, node: @cur_node)
+    send_file @item.tsv.path, type: @item.content_type, filename: @item.tsv.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/dataset/resources_controller.rb]: __[brakeman]__
* Parameter value used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/opendata/agents/nodes/app/app_controller.rb [hunk 11]
```diff
@@ -11,159 +11,159 @@ class Opendata::Agents::Nodes::App::AppController < ApplicationController
...
-    def text
-      @cur_node.layout = nil
-      if @appfile.present?
-        send_file @appfile.file.path, :type => "text/plain", filename: @appfile.filename,
-          disposition: :inline, x_sendfile: true
-      end
+  def text
+    @cur_node.layout = nil
+    if @appfile.present?
+      send_file @appfile.file.path, :type => "text/plain", filename: @appfile.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/app/app_controller.rb]: __[brakeman]__
* Parameter value used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/fs/files_controller.rb [hunk 12]
```diff
@@ -8,77 +8,77 @@ class Fs::FilesController < ApplicationController
...
 
-      if Fs.mode == :file && Fs.file?(@item.path)
-        send_file @item.path, type: @item.content_type, filename: @item.filename,
-          disposition: :inline, x_sendfile: true
-      else
-        send_data @item.read, type: @item.content_type, filename: @item.filename,
-          disposition: :inline
-      end
+    if Fs.mode == :file && Fs.file?(@item.path)
+      send_file @item.path, type: @item.content_type, filename: @item.filename,
```
Comment (sunny4381) [inline app/controllers/fs/files_controller.rb]: __[brakeman]__
* Model attribute used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/uploader/files_controller.rb [hunk 13]
```diff
@@ -1,169 +1,169 @@
...
+    raise "400"
+  end
+
+  public
+  def file
+    action = params[:do] || "index"
+    raise "404" unless @item
+    raise "404" unless %w(index new_directory new_files show edit delete).index(action)
+    send action
+  end
```
Comment (sunny4381) [inline app/controllers/uploader/files_controller.rb]: __[brakeman]__
* User controlled method execution [:ref](http://brakemanscanner.org/docs/warning_types/dangerous_send/)

File: app/controllers/opendata/agents/nodes/app/app_controller.rb [hunk 14]
```diff
@@ -11,159 +11,159 @@ class Opendata::Agents::Nodes::App::AppController < ApplicationController
...
-    def app_index
-      @cur_node.layout = nil
-      if @appfile.present?
-        send_file @appfile.file.path, type: @appfile.content_type, filename: @appfile.filename,
-          disposition: :inline, x_sendfile: true
-      end
+  def app_index
+    @cur_node.layout = nil
+    if @appfile.present?
+      send_file @appfile.file.path, type: @appfile.content_type, filename: @appfile.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/app/app_controller.rb]: __[brakeman]__
* Parameter value used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/opendata/dataset/url_resources_controller.rb [hunk 15]
```diff
@@ -10,41 +10,41 @@ class Opendata::Dataset::UrlResourcesController < ApplicationController
...
+  end
+
+  def create
+    @item = @dataset.url_resources.new get_params
+    render_create @item.save
+  end
+
+  def download
+    @item = @dataset.url_resources.find params[:url_resource_id]
+    send_file @item.file.path, type: @item.content_type, filename: File.basename(@item.filename),
```
Comment (sunny4381) [inline app/controllers/opendata/dataset/url_resources_controller.rb]: __[brakeman]__
* Parameter value used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/opendata/app/appfiles_controller.rb [hunk 16]
```diff
@@ -10,45 +10,45 @@ class Opendata::App::AppfilesController < ApplicationController
...
+  def update
+    @item.attributes = get_params
+    @item.status = params[:item][:state]
+    @item.workflow = { workflow_reset: true } if @app.member.present?
+    render_update @item.update
+  end
+
+  def download
+    @item = @app.appfiles.find params[:appfile_id]
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/app/appfiles_controller.rb]: __[brakeman]__
* Parameter value used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/cms/apis/nodes_controller.rb [hunk 17]
```diff
@@ -7,20 +7,20 @@ class Cms::Apis::NodesController < ApplicationController
...
-      return unless model.include?('::Node')
+  def set_model_from_param
+    model = params[:model].presence
+    return if model.blank?
+    return unless model.include?('::Node')
 
-      model = model.constantize rescue nil
-      return if model.blank?
-      return unless model.ancestors.include?(Cms::Model::Node)
+    model = model.constantize rescue nil
```
Comment (sunny4381) [inline app/controllers/cms/apis/nodes_controller.rb]: __[brakeman]__
* Unsafe reflection method constantize called with parameter value [:ref](http://brakemanscanner.org/docs/warning_types/remote_code_execution/)

File: app/controllers/opendata/agents/nodes/app/appfile_controller.rb [hunk 18]
```diff
@@ -6,43 +6,43 @@ class Opendata::Agents::Nodes::App::AppfileController < ApplicationController
...
-    def json
-      @cur_node.layout_id = nil
+  def json
+    @cur_node.layout_id = nil
 
-      @item = @app.appfiles.find_by id: params[:id], format: "JSON"
+    @item = @app.appfiles.find_by id: params[:id], format: "JSON"
 
-      @json = File.read(@item.file.path, :encoding => Encoding::UTF_8)
+    @json = File.read(@item.file.path, :encoding => Encoding::UTF_8)
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/app/appfile_controller.rb]: __[brakeman]__
* Model attribute used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/cms/agents/tasks/nodes_controller.rb [hunk 19]
```diff
@@ -5,59 +5,59 @@ class Cms::Agents::Tasks::NodesController < ApplicationController
...
+
+    ids.each do |id|
+      node = Cms::Node.site(@site).and_public.where(id: id).first
+      next unless node
+      next unless node.public?
+      next unless node.public_node?
+
+      node  = node.becomes_with_route
+      cname = node.route.sub("/", "/agents/tasks/node/").camelize.pluralize + "Controller"
+      klass = cname.constantize rescue nil
```
Comment (sunny4381) [inline app/controllers/cms/agents/tasks/nodes_controller.rb]: __[brakeman]__
* Unsafe reflection method constantize called with model attribute [:ref](http://brakemanscanner.org/docs/warning_types/remote_code_execution/)

File: app/controllers/opendata/dataset/resources_controller.rb [hunk 20]
```diff
@@ -10,95 +10,95 @@ class Opendata::Dataset::ResourcesController < ApplicationController
...
+    @item.attributes = get_params
+    @item.status = params[:item][:state]
+    @item.workflow = { workflow_reset: true } if @dataset.member.present?
+    render_update @item.update
+  end
+
+  def download
+    raise "403" unless @dataset.allowed?(:read, @cur_user, site: @cur_site, node: @cur_node)
+    @item = @dataset.resources.find params[:resource_id]
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/dataset/resources_controller.rb]: __[brakeman]__
* Parameter value used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/opendata/agents/nodes/dataset/url_resource_controller.rb [hunk 21]
```diff
@@ -6,35 +6,35 @@ class Opendata::Agents::Nodes::Dataset::UrlResourceController < ApplicationContr
...
+  def index
+    redirect_to @dataset_path
+  end
+
+  def download
+    @item = @dataset.url_resources.find_by id: params[:id], filename: params[:filename].force_encoding("utf-8")
+    @item.dataset.inc downloaded: 1
+
+    @cur_node.layout_id = nil
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/dataset/url_resource_controller.rb]: __[brakeman]__
* Model attribute used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/opendata/agents/nodes/mypage/dataset/my_dataset/resources_controller.rb [hunk 22]
```diff
@@ -13,135 +13,135 @@ class Opendata::Agents::Nodes::Mypage::Dataset::MyDataset::ResourcesController <
...
+    @item = @dataset.resources.find params[:resource_id]
+
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
+      disposition: :attachment, x_sendfile: true
+  end
+
+  def download_tsv
+    @item = @dataset.resources.find params[:resource_id]
+
+    send_file @item.tsv.path, type: @item.tsv.content_type, filename: @item.tsv.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/mypage/dataset/my_dataset/resources_controller.rb]: __[brakeman]__
* Parameter value used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/concerns/sns/login_filter.rb [hunk 23]
```diff
@@ -2,34 +2,34 @@ module Sns::LoginFilter
...
-      elsif params[:ref] =~ /^\//
-        redirect_to params[:ref]
-      else
-        render "sns/login/redirect"
-      end
+  def login_success
+    if params[:ref].blank?
+      redirect_to SS.config.sns.logged_in_page
+    elsif params[:ref] =~ /^\//
+      redirect_to params[:ref]
```
Comment (sunny4381) [inline app/controllers/concerns/sns/login_filter.rb]: __[brakeman]__
* Possible unprotected redirect [:ref](http://brakemanscanner.org/docs/warning_types/redirect/)

File: app/controllers/opendata/dataset/csv2rdf_settings_controller.rb [hunk 24]
```diff
@@ -16,175 +16,175 @@ class Opendata::Dataset::Csv2rdfSettingsController < ApplicationController
...
-    def render_with(opts)
-      unless request.post?
-        if opts.key?(:file)
-          render(file: opts[:file])
-        else
-          render
-        end
-        return
+    @item.attributes = get_params
+    @item.send("validate_#{params[:action]}")
```
Comment (sunny4381) [inline app/controllers/opendata/dataset/csv2rdf_settings_controller.rb]: __[brakeman]__
* User controlled method execution [:ref](http://brakemanscanner.org/docs/warning_types/dangerous_send/)

File: app/controllers/opendata/agents/nodes/app/app_controller.rb [hunk 25]
```diff
@@ -11,159 +11,159 @@ class Opendata::Agents::Nodes::App::AppController < ApplicationController
...
-      @items = pages.order_by(released: -1).limit(100)
-      render_rss @cur_node, @items
-    end
-
-    def show_point
-      @cur_node.layout = nil
-      @mode = nil
+  def download
+    zipfilename = @app.create_zip
+    send_file zipfilename, type: "application/zip", filename: "#{@app.name}.zip",
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/app/app_controller.rb]: __[brakeman]__
* Model attribute used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/opendata/agents/nodes/mypage/app/my_app/appfiles_controller.rb [hunk 26]
```diff
@@ -13,130 +13,130 @@ class Opendata::Agents::Nodes::Mypage::App::MyApp::AppfilesController < Applicat
...
 
-    def download
-      @item = @app.appfiles.find(params[:appfile_id])
+  def download
+    @item = @app.appfiles.find(params[:appfile_id])
 
-      send_file @item.file.path, type: @item.content_type, filename: @item.filename,
-        disposition: :attachment, x_sendfile: true
-    end
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/mypage/app/my_app/appfiles_controller.rb]: __[brakeman]__
* Parameter value used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/models/kana/dictionary.rb [hunk 27]
```diff
@@ -97,45 +97,45 @@ def search(params)
...
+      mecab_dicdir = SS.config.kana.mecab_dicdir
 
-        cmd = "#{mecab_indexer} -d #{mecab_dicdir} -u #{output_file} -f UTF-8 -t UTF-8 #{input_file}"
-        logger.info("system(#{cmd})")
-        system(cmd)
-        raise I18n.t("kana.build_fail.index") if $CHILD_STATUS.exitstatus != 0
-      end
+      cmd = "#{mecab_indexer} -d #{mecab_dicdir} -u #{output_file} -f UTF-8 -t UTF-8 #{input_file}"
+      logger.info("system(#{cmd})")
+      system(cmd)
```
Comment (sunny4381) [inline app/models/kana/dictionary.rb]: __[brakeman]__
* Possible command injection [:ref](http://brakemanscanner.org/docs/warning_types/command_injection/)

File: app/controllers/uploader/files_controller.rb [hunk 28]
```diff
@@ -1,169 +1,169 @@
...
+  def update
+    set_params(:filename, :files, :text, :file)
+    raise "403" unless @cur_node.allowed?(:edit, @cur_user, site: @cur_site)
+    raise "400" unless @filename
+
+    if !@item.directory?
+      @text ? (@item.text = @text) : @item.read
+    end
+    @item.filename = @filename if @filename && @filename =~ /^#{@cur_node.filename}/
+    @item.site = @cur_site
```
Comment (sunny4381) [inline app/controllers/uploader/files_controller.rb]: __[brakeman]__
* Model attribute used in regex [:ref](http://brakemanscanner.org/docs/warning_types/denial_of_service/)

File: app/controllers/opendata/agents/nodes/dataset/resource_controller.rb [hunk 29]
```diff
@@ -6,36 +6,36 @@ class Opendata::Agents::Nodes::Dataset::ResourceController < ApplicationControll
...
+  def download
+    @item = @dataset.resources.find_by id: params[:id], filename: params[:filename].force_encoding("utf-8")
+    @item.dataset.inc downloaded: 1
 
-      @cur_node.layout_id = nil
-      send_file @item.file.path, type: @item.content_type, filename: @item.filename,
-        disposition: :attachment, x_sendfile: true
-    end
+    @cur_node.layout_id = nil
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/dataset/resource_controller.rb]: __[brakeman]__
* Model attribute used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/uploader/files_controller.rb [hunk 30]
```diff
@@ -1,169 +1,169 @@
...
+  end
+
+  def redirect_from_index
+    return unless params[:filename].blank?
+    redirect_to action: :file, filename: @cur_node.filename
+  end
+
+  def set_item
+    filename = ::CGI.unescape params[:filename]
+    raise "404" if filename != @cur_node.filename && filename !~ /^#{@cur_node.filename}\//
```
Comment (sunny4381) [inline app/controllers/uploader/files_controller.rb]: __[brakeman]__
* Model attribute used in regex [:ref](http://brakemanscanner.org/docs/warning_types/denial_of_service/)

File: app/controllers/opendata/agents/nodes/mypage/dataset/my_dataset/resources_controller.rb [hunk 31]
```diff
@@ -13,135 +13,135 @@ class Opendata::Agents::Nodes::Mypage::Dataset::MyDataset::ResourcesController <
...
+  end
+
+  def show
+    render
+  end
+
+  def download
+    @item = @dataset.resources.find params[:resource_id]
+
+    send_file @item.file.path, type: @item.content_type, filename: @item.filename,
```
Comment (sunny4381) [inline app/controllers/opendata/agents/nodes/mypage/dataset/my_dataset/resources_controller.rb]: __[brakeman]__
* Parameter value used in file name [:ref](http://brakemanscanner.org/docs/warning_types/file_access/)

File: app/controllers/concerns/cms/public_filter/layout.rb [hunk 32]
```diff
@@ -10,177 +10,177 @@ module Cms::PublicFilter::Layout
...
-    def render_layout(layout)
-      @cur_layout = layout
-      @cur_item   = @cur_page || @cur_node
+    @cur_part = nil
+    body
+  end
 
-      @window_name = @cur_site.name
-      @window_name = "#{@cur_item.name} - #{@cur_site.name}" if @cur_item.filename != "index.html"
+  def render_layout(layout)
```
Comment (sunny4381) [inline app/controllers/concerns/cms/public_filter/layout.rb]: __[rubocop]__
* Assignment Branch Condition size for render_layout is too high. [79.76/68] [:ref](http://c2.com/cgi/wiki?AbcMetric)

File: app/controllers/concerns/cms/generator_filter/rss.rb [hunk 33]
```diff
@@ -2,40 +2,40 @@ module Cms::GeneratorFilter::Rss
   extend ActiveSupport::Concern
 
   private
-    def generate_node_rss(node, opts = {})
-      path = opts[:url] || "#{node.filename}/index.html"
-      return if Cms::Page.site(node.site).and_public.filename(path).first
+  def generate_node_rss(node, opts = {})
```
Comment (sunny4381) [inline app/controllers/concerns/cms/generator_filter/rss.rb]: __[rubocop]__
* Perceived complexity for generate_node_rss is too high. [15/14]

File: app/controllers/workflow/pages_controller.rb [hunk 34]
```diff
@@ -6,186 +6,186 @@ class Workflow::PagesController < ApplicationController
...
-    def fix_params
-      { cur_user: @cur_user, cur_site: @cur_site, cur_node: false }
-    end
-
-    def request_approval
+    @item.set_workflow_approver_state_to_request
+    @item.update
+  end
+
+  def workflow_alert
```
Comment (sunny4381) [inline app/controllers/workflow/pages_controller.rb]: __[rubocop]__
* Perceived complexity for workflow_alert is too high. [15/14]

File: app/controllers/workflow/pages_controller.rb [hunk 35]
```diff
@@ -6,186 +6,186 @@ class Workflow::PagesController < ApplicationController
...
+      request_approval
+      render json: { workflow_state: @item.workflow_state }
+    else
+      render json: @item.errors.full_messages, status: :unprocessable_entity
     end
+  end
 
-    def approve_update
-      raise "403" unless @item.allowed?(:approve, @cur_user)
+  def approve_update
```
Comment (sunny4381) [inline app/controllers/workflow/pages_controller.rb]: __[rubocop]__
* Perceived complexity for approve_update is too high. [16/14]

File: app/controllers/cms/preview_controller.rb [hunk 36]
```diff
@@ -64,143 +64,143 @@ def form_preview
...
-      action   = fs_path[:action]
-      size     = fs_path[:size]
-      filename = fs_path[:filename]
-      width    = params[:width]
-      height   = params[:height]
-
-      @item = SS::File.find_by(id: id_path.delete("/"), filename: filename) rescue nil
-      raise "404" unless @item
-      if width.present? && height.present?
+  def x_sendfile(file = @file)
```
Comment (sunny4381) [inline app/controllers/cms/preview_controller.rb]: __[rubocop]__
* Cyclomatic complexity for x_sendfile is too high. [15/14]
* Perceived complexity for x_sendfile is too high. [17/14]

**Comment by tany** [comment]
修飾子の下、空行あけないんですか？

**Comment by tany** [comment]
Railsの規約は下げるみたいです
https://github.com/rails/rails/blob/master/.rubocop.yml#L52

**Comment by ShinjiTanimoto** [comment]
Style/EmptyLinesAroundAccessModifierというCopで
修飾子の下の空行について設定できます。
空行を空けたほうが見た目がよくなるならば、修正します。

**Pull request merged**